src/state.rs

```diff
@@ -177,14 +177,28 @@ impl State {
             }
             KeyPress {
                 keysym: keysyms::XKB_KEY_Return,
+                ctrl,
                 ..
             } => {
                 let info = EvalInfo {
                     index: self.filtered_lines.index(self.selected_item),
                     subindex: self.selected_subitem,
                     input_value: self.input_buffer.parsed_input(),
                 };
-                self.inner.eval(info);
+                if ctrl {
+                    match unsafe { nix::unistd::fork() } {
```

Comment:
Not sure it's an intuitive place for this logic, let's move it to `impl Mode { fn fork_eval(..) }` so here is only `if ctrl { self.inner.fork_eval(..) } else { self.inner.eval(..) }`

Reply:
> Not sure it's an intuitive place for this logic

Indeed, it is unintuitive place. 

> `impl Mode { fn fork_eval(..) }`

That's way better. Thanks!